app/src/common/docs/ComponentEditor.tsx

```diff
@@ -40,15 +40,15 @@ export class ComponentEditor extends React.Component<ComponentEditorProps, Compo
                 svc.uuiNotifications
                     .show(
                         () => (
-                            <Panel background="white" shadow={ true }>
-                                <FlexRow padding="12" borderBottom={ true }>
-                                    <pre>
+                            <Panel background="surface" shadow={ true }>
+                                <FlexRow padding="24" vPadding="12">
+                                    <Text font="regular">
```

Comment:
It's better to use here monospaced font, because we display some code here. May be we can use here RTV with 'pre'?

Reply:
discussion needed